mob.go

```diff
@@ -454,6 +455,92 @@ func startTimer(timerInMinutes string, configuration config.Configuration) {
 	say.Info("It's now " + currentTime() + ". " + fmt.Sprintf("%d min timer ends at approx. %s", timeoutInMinutes, timeOfTimeout) + ". Happy collaborating! :)")
 }
 
+func abortRunningTimers() {
+	processIds := findMobTimerProcessIds()
+	for _, processId := range processIds {
+		killRunningProcess(processId)
+		say.Debug("Killed mob timer with PID" + processId)
+	}
+}
+
+func killRunningProcess(processId string) {
+	var err error
+	switch runtime.GOOS {
+	case "darwin", "linux":
+		err = killRunningProcessLinuxAndDarwin(processId)
+		break
+	case "windows":
+		err = killRunningProcessWindows(processId)
+	}
+	if err != nil {
+		say.Error(fmt.Sprintf("old timer couldn't be aborted on your system (%s)", runtime.GOOS))
+		say.Error(err.Error())
+	}
+	say.Debug("Killed mob timer with PID" + processId)
+}
+
+func killRunningProcessLinuxAndDarwin(processId string) error {
+	_, _, err := runCommandSilent("kill", processId)
+	return err
+}
+
+func killRunningProcessWindows(processId string) error {
+	_, _, err := runCommandSilent("powershell", "-command", "Stop-Process", "-Id", processId)
+	return err
+}
+
+func findMobTimerProcessIds() []string {
+	say.Debug("find all mob timer processes")
+	switch runtime.GOOS {
+	case "darwin", "linux":
+		return findMobTimerProcessIdsLinuxAndDarwin()
+	case "windows":
+		return findMobTimerProcessIdsWindows()
+	}
+	return nil
+}
+
+func findMobTimerProcessIdsWindows() []string {
```

Comment:
Where you able to test that on a Windows computer?

Reply:
I tested it manually and also used windows to develop the windows part. But I also thought about if there is a possibility to run the tests in our pipeline once on a linux machine and on a windows machine. I will do some research in the next days.

Reply:
I think we need to take the idea of automated tests for windows into a new issue, because we need to get the create testbed running on windows as described in #271. I think we should rewrite the create testbed in go, such that it works on every os.

Reply:
Okay

Reply:
I think rewriting it in Go is great, but not because of running on being able to run on windows (which can be achivied probably much easier), but because we can have different testbeds more easily. Most tests require a simple testbed, but some tests require a more sophisticated testbed. At the moment, we have one testbed for every test, and thus, testing takes longer than it should take.

Reply:
I agree